src/R/Editor/Impl/Extensions/AstRootExtensions.cs

```diff
@@ -208,7 +208,7 @@ public static class AstRootExtensions {
         /// </summary>
         public static bool IsInObjectMemberName(this ITextProvider textProvider, int position) {
             if (position > 0) {
-                for (var i = position - 1; i >= 0; i--) {
+                for (var i = position - 1; i >= 0 && i < textProvider.Length; i--) {
```

Comment:
`i` doesn't increment in the loop. So is it `textProvider.Length` that changes?

Reply:
No, when `textProvider.Length == 0` it throws when position is 1. Basically should not be dependent in position value.